### PR TITLE
"Metroidvania" patch

### PR DESCRIPTION
**Pull request by DavidMag**

### CODE REVIEW

**Comment by DavidMag** [comment]
@asdBeowulf Review this please <3

**Comment by DavidMag** [comment]
@lrgilbert Hi again! Made some changes in my fork, if you like you could check them and if you like this I can maybe make these changes optional based on the ldtk map type you are using (Gridvania for this and in other cases use the old import system).

Check the updated README for an introduction

File: addons/LDtk-Importer/LDtk.gd [hunk 1]
```diff
@@ -73,30 +78,29 @@ func new_entity(entity_data, level, options):
...
 		if data['name'] in new_entity:
 			new_entity[data['name']] = data['value']
 		else:
 			new_entity.set_meta(data['name'], data['value'])
-
+	
 	if is_custom_entity:
 		return new_entity
-
+
```
Comment (asdBeowulf) [inline addons/LDtk-Importer/LDtk.gd:93]: empty lines should be empty

**Review by asdBeowulf — APPROVED** [review]

File: addons/LDtk-Importer/LDtk.gd (hunk 1 above)
Comment (DavidMag) [reply addons/LDtk-Importer/LDtk.gd:93]: Should they really be empty here since it's the same indentation?

Comment (asdBeowulf) [reply addons/LDtk-Importer/LDtk.gd:93]: Maybe not, not really that important. Do what you think is better.

**Pull request merged**